electricpy/__init__.py

```diff
@@ -4485,42 +4682,95 @@ def characterz(R,G,L,C,freq=60):
                 Capacitance in Farads.
     freq:       float, optional
                 System frequency in Hz, default=60
-    
+
     Returns
     -------
     Zc:         complex
                 Charcteristic Impedance of specified line.
     """
     # Evaluate omega
-    w = 2*_np.pi*freq
+    w = 2 * _np.pi * freq
     # Evaluate Zc
-    Zc = _np.sqrt((R+1j*w*L)/(G+1j*w*C))
-    return(Zc)
+    Zc = _np.sqrt((R + 1j * w * L) / (G + 1j * w * C))
+    return (Zc)
+
+#define propagation_constants for long transmission line
+def propagation_constants(z, y, length):
+    
+    '''
```

Comment:
I think this is a good start for the documentation! I like to format the math in a LaTeX-supported format so that Sphinx can build it into "pretty" mathematic formulas for the web-docs, I also like to identify the data types of the inputs/outputs.

What do you think about the following:

```
    """
    Transaction Line Propagation Constant Calculator

    This function will evaluate the propagation constants for a long transmission
    line whose properties are governed by the differential equation:
    
    .. math:: \\frac{d^2V}{dx^2} = \\gamma V
    
    From the above equation, the following formulas are derived to evaluate the
    desired constants.
    
    .. math:: \\gamma = \\sqrt( z * y )
    
    .. math:: Z_{\\text{surge}} = \\sqrt( z / y )
    
    .. math:: \\alpha = \\Re{ \\gamma }
    
    .. math:: \\beta = \\Im{ \\gamma }
    
    Parameters
    ----------
    z:              complex
                    Impedence of the transmission line: R+j*2*pi*f*L
    y:              complex
                    Admitance of the transmission line g+j*2*pi*f*C
        
    Returns
    -------
    params:    dict
               Dictionary of propagation constants including:
                    
                         gamma:   Propagation constant
                         zc:            Surge impedance
                         alpha:      Attenuation constant
                         beta:        Imaginary portion of gamma
"""
````